compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -2061,8 +2066,7 @@ private BType getTypedBindingPatternTypeForXmlCollection(BType collectionType) {
                 Set<BType> collectionTypes = getEffectiveMemberTypes((BUnionType) constraint);
                 Set<BType> builtinXMLConstraintTypes = getEffectiveMemberTypes
```

Comment:
The `builtinXMLConstraintTypes` variable is never used, correct? If so, we can remove that variable.